src/main/kotlin/com/gatecheck/gatecheck/service/RequestService.kt

```diff
@@ -0,0 +1,16 @@
+package com.gatecheck.gatecheck.service
+
+import com.gatecheck.gatecheck.dao.request.RequestDao
+import com.gatecheck.gatecheck.model.Request
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Service
+import java.util.*
+
+
+@Service
+class RequestService @Autowired construct(@Qualifier("requestDao") private val requestDao:RequestDao){
```

Comment:
Qualifier name should match repository name which is a great reason as to why consistency is important hahaha